config/locales/en.yml

```diff
@@ -19,6 +19,13 @@ en:
       not_confirmed: "A confirmation email was sent to your account at '%{email}'. You must follow the instructions in the email before your account can be activated"
     confirmations:
       invalid_token: "Invalid confirmation token. Please try again"
+      user_not_found: "Unable to find user with email '%{email}'."
+      sended: "An email has been sent to '%{email}' containing instructions for confirming your account."
+      send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
```

Comment:
Son only this one and `user_not_found ` are used in your PR. The other are not.

Reply:
I removed the extra ones. If they need to use Devise's extra "paranoid" text, they can copy it to the `send_instructions` entry or override this method in their own application.